### DISCUSSION TITLE
Display spec for functions when it's defined and reachable?

### DISCUSSION THREAD

**Discussion by bsless** [borkdude/quickdoc#13] [Ideas]
Title pretty much says it all,
Instead of having to repeat myself in specs and textual description of options, can the specs for a `fdef`ed function be collected and displayed after the docstring?

Already using quickdoc, great stuff :)

**Comment by borkdude** [comment]
Thanks!

I think that would be interesting. This would mean that clj-kondo would gather this information into the analysis. Perhaps this could also be useful for clojure-lsp to display things. 

I think we had a related issue for schema somewhere in clj-kondo, @ericdallo, do you remember which issue that was?

**Reply by ericdallo** [reply]
Yes, it seems it'd help with that
https://github.com/clj-kondo/clj-kondo/issues/1268

**Reply by borkdude** [reply]
OK, thanks for digging that up, I re-opened the issue.
